src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -299,6 +320,13 @@ def bot_equiv : (⊥ : subgraph G).coe ≃g (⊥ : simple_graph empty) :=
   right_inv := λ v, v.elim,
   map_rel_iff' := λ a b, iff.rfl }
 
+lemma edge_set_mono {H₁ H₂ : subgraph G} (h : H₁ ≤ H₂) : H₁.edge_set ≤ H₂.edge_set :=
+λ e, sym2.ind h.2 e
+
+lemma disjoint.subgraph_edge_set {H₁ H₂ : subgraph G}
```

Comment:
If this is for dot notation, maybe this works better? (Didn't verify myself.)
```suggestion
lemma _root_.disjoint.subgraph_edge_set {H₁ H₂ : subgraph G}
```

Reply:
It does, thanks!